docs/BUILD.bazel

```diff
@@ -17,6 +17,18 @@ stardoc(
     deps = ["//nixpkgs"],
 )
 
+stardoc(
+    name = "core",
```

Comment:
Shouldn't there also be stardocs and generated READMEs for the other components?
Side note: Happy to leave the docs aspect for a follow-up PR.

Reply:
Prepared that step and opened #197.